hw/ip/flash_ctrl/doc/_index.md

```diff
@@ -425,7 +425,7 @@ When disabled, the flash has a two layered response:
 - The flash protocol controller [memory protection]({{< relref "#memory-protection" >}}) errors back all controller initiated operations.
 - The host-facing tlul adapter errors back all host initiated operations.
 - The flash physical controller completes any existing stateful operations (program or erase) and drops all future flash transactions.
-- The flash protocol controller arbiter completes any existing software issued commands and enters an invalid state where no new transactions can be issued.
+- The flash protocol controller arbiter completes any existing software issued commands and enters a disabled state where no new transactions can be issued.
```

Comment:
Maybe it makes sense to point out the difference in behavior between escalation  and disable either here or in the `Flash Escalation` section above (e.g. sending out alerts) ?

Reply:
actually this part is still the same. The escalation also brings the module into disable. Let me try to make that clearer. The main difference now is that escalation does not cause alerts, only explicit local errors cause alerts, which can lead to escalation.

Reply:
Oh I see, thanks for the clarification.